pkg/kubelet/kuberuntime/kuberuntime_container_windows_test.go

```diff
@@ -101,3 +98,48 @@ func TestApplyPlatformSpecificContainerConfig(t *testing.T) {
 	}
 	assert.Equal(t, expectedWindowsConfig, containerConfig.Windows)
 }
+
+func TestCalculateCPUMaximum(t *testing.T) {
```

Comment:
Thanks for adding tests to cover this section!  It looks like there was some coverage in the test above but this adds coverage for cpucounts greater than 64. In particular might need to change line 87 `expectedCpuMax := ((10000 * 3000) / int64(runtime.NumCPU()) / 1000)` or that would fail if run on a larger machine

Reply:
yes please

Reply:
adjusted 👍